src/flexbox/nasa.css

```diff
@@ -0,0 +1,72 @@
+body {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-around;
+    height: 100%;
+    width: 100%;
+/*  flex settings affecting div & main */
+}
+
+div {
+  height: 58%;
+  width: 100%;
+  margin-bottom: 10px;
+  background-image: url(./galaxy.png); 
+  /* want this img fit into header container
+  ended up using div + set body container h&w to 100% */
+  text-align: center;  /* need to justify & align text for child elem of flex container */
```

Comment:
yes. this is exactly how you do this. image background with text element in middle.
since this is a flexbox assignment, lets use flexbox here. remove `text-align` on this div. add `display: flex; justify-content:center;align-items:center;`. you'll also have to remove `margin: 13%` on the p tag `#nasatitle`